src/Makefile.am

```diff
@@ -42,7 +42,7 @@ libcalf_la_LIBADD = $(FLUIDSYNTH_DEPS_LIBS) $(GLIB_DEPS_LIBS)
 if USE_DEBUG
 libcalf_la_LDFLAGS = -rpath $(pkglibdir) -avoid-version -module -lexpat -disable-static
 else
-libcalf_la_LDFLAGS = -rpath $(pkglibdir) -avoid-version -module -lexpat -disable-static -export-symbols-regex "lv2_descriptor"
```

Comment:
If both branches (`if`, `else`) are equal, I guess we can unify this 5-line-construct into one?

Reply:
Yeah, you're right. Didn't notice this made them equal. Will do.